modules/ROOT/pages/cloudhub-architecture.adoc

```diff
@@ -74,14 +90,17 @@ Each worker is a dedicated instance of Mule that runs your integration applicati
 * 8 vCores + 15 GB Heap Memory
 * 16 vCores + 32 GB Heap Memory
 
+For workers that use a fraction of a vCore, CloudHub allocates limited CPU and IO as well as less memory. 
+Use these worker sizes for apps with smaller workloads.
 
-[NOTE]
-Along with less available memory, workers that use a https://aws.amazon.com/ec2/instance-types/#burst[fraction of a vCore] have more limited CPU and IO, which are usually only appropriate for smaller workloads.
+For apps deployed to CloudHub with worker sizes of 0.1, 0.2, and 1 vCore, the metaspace limit is 256 MB.
```

Comment:
For the current documentation, we should just mention that for apps deployed to Cloudhub, the metaspace limit is 256MB.